### DISCUSSION TITLE
Solution for closed loop

### DISCUSSION THREAD

**Discussion by ZackeyZAKI** [OpenModelica/OpenModelica#10721] [Q&A] [answered]
In OMEdit, when simulating a closed loop Fluid model, an error occurs. 
What elements should be added to the circuit to address this? (The role of GND in electrical circuits)

**Comment by casella** [comment, marked as answer]
A fixed pressure source would do. If the fluid is incompressible, there will be zero flow in it as a result. If it is nearly incompressible, it will be near zero.
